pages/docs/project-management/auto-archive.mdx

```diff
@@ -0,0 +1,32 @@
+---
+description: Auto archive allow you to automatically move to trash resources that have not been viewed based on your settings.
+---
+
+import { Callout } from "nextra-theme-docs";
+
+# Auto archive
+
+Auto archive allow you to automatically move to trash resources that have not been viewed based on your settings.
+
+## Settings
+
+- Days: Set the time frame for automatically moving unused resources to trash. By default, resources are moved after 30 days of inactivity.
```

Comment:
Let's be more specific the just `inactivity`. Maybe something like "after 30 days without any views by users of your project."

Reply:
👍